integrations/tests/test_maltiverse.py

```diff
@@ -0,0 +1,422 @@
+from socket import AF_UNIX, SOCK_DGRAM
```

Comment:
```suggestion
# Copyright (C) 2015, Wazuh Inc.
# Created by Wazuh, Inc. <info@wazuh.com>.
# This program is free software; you can redistribute
# it and/or modify it under the terms of GPLv2

from socket import AF_UNIX, SOCK_DGRAM
```

Reply:
Added comments in commit a5daea01dc774d489200de66f0966905e34c272c.